scripts/prod_vs_int.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+# Each time that this script is executed, this script does the following: 
+# - One commit is done for the copy of the files at production phase
+# - One commit is done for the copy of the same files at integration phase
+# - The output github link shows the differences between production-integration phase for each file
+
+set -e
+
+# Output color of echo
+black=`tput setaf 0`
+red=`tput setaf 1`
+reset=`tput sgr0`
+background=`tput setab 7`
+
+# **************************** !!IMPORTANT!! *************************************
+# THE FIRST TIME BEFORE EXECUTING THE SCRIPT :
+# (re-)initialize Git repository & clone the <prod_vs_int_review> branch of the db project from github
+# git init
+# git clone -b prod_vs_int_review --single-branch git@github.com:geoadmin/db.git 
+# ********************************************************************************
+
+# If permission is denied when executing the bash script, do: chmod u+x <name>
+# At the <prod_vs_int_preview> branch all the changes are commited between the different phases (production - integration phases)
+# for different dates (a history of changes between the different phases is being kept)
+git stash
+cd ../db
+git checkout prod_vs_int_review
+
+# Now we are inside geoadmin/db and here the script is executable 
+# mkdir -p prod_vs_int_review 
+cd prod_vs_int_review
+
+# Date and time of executing the script
+now=$(date +%Y-%m-%d--%H:%M:%S)
+prodJsons="https://api3.geo.admin.ch/rest/services/all/MapServer/layersConfig|allLayersConfig.json
```

Comment:
can we get the base URLs direclty from `rc_int` ? Or at least define them once and reuse them later.